tests/test_GaussianSourceCalculator.py

```diff
@@ -0,0 +1,19 @@
+"""Test EMCPhoton"""
```

Comment:
test module filenames should be TestGaussianSourceCalculator.py (no underscore, first letter Capitalized)